.kitchen.dokken.yml

```diff
@@ -63,6 +69,10 @@ platforms:
   driver:
     image: dokken/opensuse-leap
     pid_one_command: /bin/systemd
+  attributes:
+    java:
+      jdk_version: '8'
+      openjdk_packages: ['java-1_8_0-openjdk']
```

Comment:
i created a pr to java cookbook to make this default properly
https://github.com/agileorbit-cookbooks/java/pull/446